doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -29,19 +29,41 @@ In order to promote the quick deployment of the RI and avoid duplicating effort
 
 <a name="1.1.3"></a>
 ### 1.1.3 Terminology
 
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md).
+
 <a name="1.2"></a>
 ## 1.2 Scope
 
+The scope of this document is as follows:
+1. To generate eco-system requirements for the establishment of the RI, including labs, tooling, installers, releases and automation requirements
+1. Provide detailed description file definition for installers, CNF features, and etc..
+1. Provide detailed lab criteria and operations that are generic enough to allow any environment to be used as the "lab".
+1. Provide an operational runbook for the RI, which includes detailed steps for the deployment and configuration of the RI into a chosen lab environment
+1. Provide detailed design for automation of deployment and testing, provide continuous integration and delivery pipeline for the RI
+1. Gap analysis with required actions for existing eco-system within adjacent community projects
+
 <a name="1.3"></a>
 ## 1.3 Reference Implementation Approach
 
-> Describe the approach taken in building the RI (e.g. combined infra/k8s LCM, or sepaarate, etc.)
+The approach taken for the RI has deliberately started by having separate automation stages for the infrastructure on which the RI will be installed, and the RI itself. Meaning, initially, the RI is not looking to have a single installer that can both build out virtual or physical machines **and** build out the Kubernetes and other components.
+
+The primary reason for this was to ensure the loose coupling between the two layers, to drive the concept that RI2 is a standalone platform that can (in theory at least) be deployable to any infrastructure, whether that be some physical machines in a lab, or virtual machines in a private or public cloud environment, for example. To combine the deployment of machines and Kubernetes platform with a single installer would potentially limit the locations on which the RI can be installed.
+
+However, that's not to say full end-to-end installers aren't welcome, but if used we must be clear on the limitations and compromises made when we document them in [Chapter 4](./chapter04.md).
 
 
 <a name="1.4"></a>
 ## 1.4 Relationship to other communities
+
+The main communities involved in driving requirements and development of this RI are:
+- **OPNFV - Cloud Native OVP aka OVP2.0**
+    - OPNFV, and CNOVP within it, owns the definition of the end-to-end framework (tooling, process, tests) used to verify and certify cloud native infrastructure and network functions
```

Comment:
terminology nit-pick: better refer to "conformance testing" instead of "certify / certification"
```suggestion
    - OPNFV, and CNOVP within it, owns the definition of the end-to-end framework (tooling, process, tests) used to verify compliance of cloud native infrastructure and network functions
```

Reply:
This was taken from the CNOVP presentation that happened in the vF2F.  I think it's right to say that CNOVP owns certification, isn't it? Given they own the badging etc.

Reply:
They absolutely own the badging and process around it. I was merely referring to the word "certification", which to my knowledge, we should not use as it is not a certification program in legal terms, but rather a compliance or conformance program.

Reply:
OK.  @xmulligan what's your opinion on this?  I took the wording from the deck you presented.  Happy to change, but want to be accurate.

Reply:
FYI: just reading through the RC2 intro which seems to carefully avoid the term "certification" ;-) 

https://github.com/cntt-n/CNTT/pull/1624/files?short_path=a5d88ac#diff-a5d88acaba38dfcb1dff1bc46ab88be3

Reply:
So I 100% agree that RC2 and anything CNTT-related should avoid certification, but this part of the doc is saying that OPNFV owns that certification, not anything CNTT-related.

Reply:
ok, got it. Still, back in the days of OVP 1.0, the same advice was given by LF on the use of "certification".

It is a terminology nit-pick in the context of this chapter and I also don't want to be overly picky here - just pointing this out. Happy to hear opinions from others .

Reply:
"certification" has legal implications and hence the use of the term "conformance" and "compliance"